src/GAP.jl

```diff
@@ -39,7 +49,48 @@ import Markdown
 """
     GapObj
 
-TODO
+This is the Julia type of all those GAP objects that are not
+"immediate" (Booleans, small integers, FFEs).
+
+# Examples
+```jldoctest
+julia> isa( GAP.EvalString( "[ 1, 2 ]" ), GAP.GapObj ) # a GAP list
+true
+
+julia> isa( GAP.EvalString( "rec()" ), GAP.GapObj )    # a GAP record
+true
+
+julia> isa( GAP.EvalString( "(1,2,3)" ), GAP.GapObj )  # a GAP permutation
+true
+
+julia> isa( GAP.EvalString( "2^64" ), GAP.GapObj )     # a large GAP integer
+true
+
+julia> isa( GAP.EvalString( "2^59" ), GAP.GapObj )     # a small GAP integer
+false
+
+julia> isa( GAP.EvalString( "Z(2)" ), GAP.GapObj )     # a GAP FFE
+false
+
+julia> isa( GAP.EvalString( "true" ), GAP.GapObj )     # a Boolean
+false
```

Comment:
Instead of showing what this is *not* we could also just show the respective types? I.e.,
```suggestion
julia> typeof( GAP.EvalString( "true" ) )     # a Boolean
Bool
```
and similar for the other cases